deno/payloads/v10/gateway.ts

```diff
@@ -152,7 +152,10 @@ export interface GatewayActivity {
 	 */
 	timestamps?: GatewayActivityTimestamps;
 	sync_id?: string;
-	platform?: ActivityPlatform;
+	/**
+	 * {@link ActivityPlatform}
```

Comment:
1. Does it work
2. You should use something more descriptive (`You can use {@link ActivityPlatform} as a stepping stone, but this might be inaccurate`)

Reply:
> but this might be inaccurate

tbh this is true for all enums since discord adds new values all the time ¯\\\_(ツ)_/¯